data/data/bootstrap/baremetal/systemd/units/master-bmh-update.service

```diff
@@ -2,6 +2,7 @@
 Description=Update master BareMetalHosts with introspection data
 Wants=bootkube.service
 After=release-image.service
+Before=progress.service
```

Comment:
progress service is supposed to be our one place that defines `done`, adding this to this service would make figuring that out a little more complex.

If we have a better way to define this in a more concise way, i would love that. but for now i wouldn't block this change.

Reply:
Thanks. Theoretically we could put `After=master-bmh-update.service` in the progess service instead - systemd treats ordering completely separately from dependencies, so it wouldn't care on platforms where this service doesn't exist. I didn't go with that initially because it seemed like it might cause more confusion on non-baremetal platforms than it solves on baremetal :) Completely open to switching it over if you prefer that though.